python/conftest.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+# Add the import here for any plugins that should be loaded EVERY TIME
+pytest_plugins = ("cuml.test.plugins.quick_run_plugin")
+
+
+def pytest_addoption(parser):
+    # Any custom option, that should be available at any time (not just a
+    # plugin), goes here.
+
+    group = parser.getgroup('cuML Custom Options')
+
+    group.addoption(
+        "--run_stress",
+        action="store_true",
+        default=False,
+        help=("Runs tests marked with 'stress'. These are the most intense "
+              "tests that take the longest to run and are designed to stress "
+              "the hardware's compute resources."))
+
+    group.addoption(
+        "--run_quality",
+        action="store_true",
+        default=False,
+        help=("Runs tests marked with 'quality'. These tests are more "
+              "computationally intense than 'unit', but less than 'stress'"))
+
+    group.addoption(
+        "--run_unit",
+        action="store_true",
+        default=False,
```

Comment:
Wasnt sure how we wanted these to work. By default, `--run_unit` is implied if neither `--run_stress` or `--run_quality` is specified. This is how it currently functions.

I can set the default to true but that would change the functionality. Currently, if you set just `--run_stress`, only the stress tests are run and unit/quality are disabled. Defaulting `--run_unit` to true just makes it hard to disable since its set as a flag. We could add additional not flags (i.e. `--no-run_unit`) to make this easier but seems more complicated than its worth.

Reply:
Keeping the current approach seems least confusing - thanks for clarification